src/main/java/com/jfrog/ide/idea/ui/configuration/JFrogGlobalConfiguration.form

```diff
@@ -246,10 +246,27 @@
           </component>
           <component id="5d973" class="com.intellij.ui.HyperlinkLabel" binding="infoPanel">
             <constraints>
-              <grid row="14" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="9" fill="0" indent="0" use-parent-layout="false"/>
+              <grid row="16" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="9" fill="0" indent="0" use-parent-layout="false"/>
             </constraints>
             <properties/>
           </component>
+          <component id="fd2cc" class="javax.swing.JLabel" binding="ssoLoginInstructionsLabel">
+            <constraints>
+              <grid row="8" column="0" row-span="1" col-span="4" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="SSO login instructions"/>
+            </properties>
+          </component>
+          <component id="a0ab" class="javax.swing.JLabel" binding="ssoCode">
```

Comment:
Let's make it a non-editable JBTextField to make it copyable.

Also, please consider rearranging it to make it as follows:
![image](https://github.com/jfrog/jfrog-idea-plugin/assets/11367982/79e0ccf6-fe87-4f2f-9cac-79397576273f)